src/GovUk.Education.ExploreEducationStatistics.Data.Api/Controllers/PublicationController.cs

```diff
@@ -18,10 +18,20 @@ public PublicationController(IPublicationService publicationService)
             _publicationService = publicationService;
         }
 
-        [HttpGet("{id}")]
-        public async Task<ActionResult<PublicationViewModel>> GetPublication(Guid id)
+        [HttpGet("publications/{publicationId}")]
+        public async Task<ActionResult<SubjectsAndHighlightsViewModel>> GetLatestPublicationSubjectsAndHighlights(Guid publicationId)
         {
-            return await _publicationService.GetPublication(id).HandleFailuresOrOk();
+            return await _publicationService
+                .GetLatestPublicationSubjectsAndHighlights(publicationId)
+                .HandleFailuresOrOk();
+        }
+
+        [HttpGet("releases/{releaseId}")]
```

Comment:
This naming within the PublicationController doesn't seem super nice to me as it's not really a Release that's returns, nor a Publication, but I suppose this is the Data API rather than the Content API so a Release can mean something slightly different here.